### ISSUE TITLE
bug: ion-datetime fires multiple ionChange events

### ISSUE THREAD

**Issue by edmirjano** [ionic-team/ionic-framework#25567]
### Prerequisites

- [X] I have read the [Contributing Guidelines](https://github.com/ionic-team/ionic-framework/blob/main/.github/CONTRIBUTING.md#creating-an-issue).
- [X] I agree to follow the [Code of Conduct](https://ionicframework.com/code-of-conduct).
- [X] I have searched for [existing issues](https://github.com/ionic-team/ionic-framework/issues) that already report this problem, without success.

### Ionic Framework Version

- [ ] v4.x
- [ ] v5.x
- [X] v6.x
- [ ] Nightly

### Current Behavior

as i am using ion-datetime component on ionic v6 like below:

` <ion-button id="open-date-input">Change</ion-button>
        <ion-popover #popover trigger="open-date-input" show-backdrop="false">
          <ng-template>
            <ion-datetime #time presentation="time" size="cover"
              [value]="selectedTime"
              (ionChange)="changeDate(time.value)" [showDefaultButtons]="true">
            </ion-datetime>
          </ng-template>
        </ion-popover>
`

i get 'changeDate' fired twice every time I press 'done; inside the popover:
<img width="170" alt="Screenshot 2022-07-02 at 3 06 49 PM" src="https://user-images.githubusercontent.com/48261411/177002101-d053022d-83c5-4026-8e53-df88306b2c31.png">

### Expected Behavior

event should be fired only once

### Steps to Reproduce

simple as i showed you before

### Ionic Info

Ionic:

   Ionic CLI                     : 6.19.0 (/usr/local/lib/node_modules/@ionic/cli)
   Ionic Framework               : @ionic/angular 6.0.13
   @angular-devkit/build-angular : 13.2.6
   @angular-devkit/schematics    : 13.2.6
   @angular/cli                  : 13.2.6
   @ionic/angular-toolkit        : 6.1.0

Capacitor:

   Capacitor CLI      : 3.4.3
   @capacitor/android : 3.4.3
   @capacitor/core    : 3.4.3
   @capacitor/ios     : 3.4.3

Utility:

   cordova-res                          : not installed globally
   native-run (update available: 1.6.0) : 1.5.0

System:

   NodeJS : v18.1.0 (/Users/edmirjano/.nvm/versions/node/v18.1.0/bin/node)
   npm    : 8.8.0
   OS     : macOS Monterey

**Comment by amandaejohnston** [comment]
This issue is being closed due to lack of a sufficient reproduction. If this is still a problem on the latest version of Ionic, please create a new issue and ensure the template is fully filled out. Thank you!
